packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -195,37 +187,60 @@ const OpenQueueCard = ({
             </Tooltip>
           ))}
         </div>
-        {editingNotes && (
-          <SaveButton onClick={handleUpdate} size="large">
-            Save Changes
-          </SaveButton>
-        )}
-        {!editingNotes && (
-          <QueueCardButtonRow>
-            {isTA && (
-              <EditNotesButton
-                size="large"
-                onClick={() => {
-                  setEditingNotes(true);
-                }}
-              >
-                Edit Notes
-              </EditNotesButton>
-            )}
-            <Link
-              href="/course/[cid]/queue/[qid]"
-              as={`/course/${cid}/queue/${queue.id}`}
+        <OpenQueueCardDivider />
+        {editingNotes ? (
+          <div>
+            <NotesInput
```

Comment:
bit of a nitpick, but I think it would look nice if this input spanned the whole width of the card (up until the `edit notes` button)

Reply:
done